src/cdef.rs

```diff
@@ -1147,7 +1153,7 @@ unsafe extern "C" fn cdef_filter_8x8_neon_erased<BD: BitDepth>(
                 dir,
                 damping,
                 8,
-                edges as usize,
+                edges.bits() as usize,
```

Comment:
I think it's fine to do the explicit cast here (as you are), since this is calling an asm `fn` and we do ptr casts for those all of the time anyways.

Reply:
I found this comment on the C declarations of these asm `fn`s:

```c
// Passing edges to this function, to allow it to switch to a more
// optimized version for fully edged cases. Using size_t for edges,
// to avoid ABI differences for passing more than one argument on the stack.
```

So doing the explicit cast here makes sense.